sql/spanner/inspect.go

```diff
@@ -214,6 +217,19 @@ func columnType(spannerType string) (schema.Type, error) {
 	col := &columnDesc{}
 	spannerType = strings.TrimSpace(strings.ToUpper(spannerType))
 
+	// ARRAY type handling.
+	if strings.HasPrefix(spannerType, "ARRAY<") {
```

Comment:
You can also use `arrayTypeRe.MatchString(spannerType )`.

Reply:
what's preferable?